clang/lib/AST/TypePrinter.cpp

```diff
@@ -2048,6 +2049,17 @@ void TypePrinter::printBTFTagAttributedAfter(const BTFTagAttributedType *T,
   printAfter(T->getWrappedType(), OS);
 }
 
+void TypePrinter::printHLSLAttributedResourceBefore(
+    const HLSLAttributedResourceType *T, raw_ostream &OS) {
+  printBefore(T->getWrappedType(), OS);
+  // FIXME: print values of resource type attributes here
```

Comment:
Yes, we can! I was not sure how many properties should I include with this PR, I was switching between none and the two  I added. Might as well print them since I've decided to include them.